tests/src/Interop/StringMarshalling/AnsiBSTR/AnsiBStrTestNative.cpp

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#include <xplatform.h>
+#include "platformdefines.h"
+
+const char *strManaged = " \0Managed\0String\0 ";
+size_t lenstrManaged = 18; // the length of strManaged
```

Comment:
This can easily be computed using the `ARRAYSIZE(strManaged) - 1;`. Note you need to change the type above to an `[]`. Same below.